notebooks/catvton/catvton.ipynb

```diff
@@ -0,0 +1,271 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        %pip install -q "numpy&lt;2.0.0"
</pre><p> this is not critical, but it is better to install numpy in the last order to guarantee that exact this version will be used in notebook (some of third-party may override that)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2453/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>